### PR TITLE
[KAJ-1201] & [KAJ-1195] & [KAJ-1193] Windows & Mac & eLux 1.7.0 Documentation

### PR DESCRIPTION
**Pull request by pegnovi**
## Story
[KAJ-1201](https://kandyio.atlassian.net/browse/KAJ-1201) Windows 1.7.0: Documentation
[KAJ-1195](https://kandyio.atlassian.net/browse/KAJ-1196) Mac 1.7.0: Documentation
[KAJ-1193](https://kandyio.atlassian.net/browse/KAJ-1193) eLux 1.7.0: Documentation

## Description
Updated Changelogs for Windows, Mac, & eLux.

### CODE REVIEW

File: eLux/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,28 @@ Kandy Distant Driver for VDI eLux change log.
...
+
+- eLux: Configuration parameter `DebugUrlEnabled` now available. `KAJ-1095`
+- eLux: Renamed kanding_distant.so  to KandyDistant.DLL. `KAJ-1210`
+
+### Fixed
+- Orchestrator process does not close after simulating browser crash. `KAJ-716`
+- Distant Reinitialization fails during Virus Scan. `KAK-1299`
+- Browser Start Success doesn't persist all session properties. `KAJ-1307`
+- Browser Message Handler alters session state. `KAJ-1303`
+- Distant Tracker: Fixed address exception when exiting fullscreen. `KAJ-1189`
```
Comment (jfmoron) [inline eLux/CHANGELOG.md:24]: you should remove tracker related jira from here

File: eLux/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,28 @@ Kandy Distant Driver for VDI eLux change log.
...
+- Reinit Commlink/Orch when heartbeat is missed. `KAJ-1247`
+- Improved multi-session support on browser process. `KAJ-1118`
+- Logs will indicate if orchestrator fails to open. `KAJ-1164`
+
+- eLux: Configuration parameter `DebugUrlEnabled` now available. `KAJ-1095`
+- eLux: Renamed kanding_distant.so  to KandyDistant.DLL. `KAJ-1210`
+
+### Fixed
+- Orchestrator process does not close after simulating browser crash. `KAJ-716`
+- Distant Reinitialization fails during Virus Scan. `KAK-1299`
```
Comment (jfmoron) [inline eLux/CHANGELOG.md:21]: typo this should be KAJ instead of KAK

File: mac/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,37 @@ Kandy Distant Driver for VDI Mac change log.
...
+
+### Added
+- Multiple session with the same id can be created. `KAJ-1034`
+- Reinit Commlink/Orch when heartbeat is missed. `KAJ-1247`
+- Improved multi-session support on browser process. `KAJ-1118`
+- Logs will indicate if orchestrator fails to open. `KAJ-1164`
+
+### Fixed
+- Orchestrator process does not close after simulating browser crash. `KAJ-716`
+- Distant Reinitialization fails during Virus Scan. `KAK-1299`
```
Comment (jfmoron) [inline mac/CHANGELOG.md:18]: KAK vs KAJ

**Review by jfmoron — CHANGES_REQUESTED** [review]

File: mac/CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,37 @@ Kandy Distant Driver for VDI Mac change log.
...
+
+- MacOS: Aux window causes unnecessary hide. `KAJ-1310`
+- MacOS: CEF steals focus from CitrixViewer upon creation. `KAJ-930`
+- MacOS: CEF steals focus from CitrixViewer when clicked. `KAJ-937`
+- MacOS: Distant Window regression - moving window while hidden does not work. `KAJ-1251`
+- MacOS: "About Citrix" auxiliary window was not being properly recognized. `KAJ-1231`
+- MacOS: Inner content is displayed at the wrong size when minimizing/unminimizing. `KAJ-1222`
+- MacOS: Distant Window now has menubar offset when in fullscreen mode. `KAJ-1112`
+- MacOS: Distant window orders it's Z-index behind Citrix preferences. `KAJ-1084`
+- MacOS: Distant window's contents are undisturbed by intersecting Citrix preferences. `KAJ-1084`
```
[the remark this answers is not in the document]
Comment (pegnovi) [reply mac/CHANGELOG.md]: They were related fixes. I have combined them into one log and summarized it.

File: eLux/CHANGELOG.md (hunk 1 above)
Comment (pegnovi) [reply eLux/CHANGELOG.md]: Removed

File: mac/CHANGELOG.md (hunk 3 above)
Comment (pegnovi) [reply mac/CHANGELOG.md]: Fixed.

File: windows/CHANGELOG.md [hunk 5]
```diff
@@ -5,10 +5,31 @@ Kandy Distant Driver for VDI Windows change log.
...
+- Reinit Commlink/Orch when heartbeat is missed. `KAJ-1247`
+- Improved multi-session support on browser process. `KAJ-1118`
+- Logs will indicate if orchestrator fails to open. `KAJ-1164`
+
+- Windows: Handle browser process not opening and log the error. `KAJ-541`
+- Windows: Executables produce a dump file on system exceptions. `KAJ-1155`
+
+### Fixed
 - Orchestrator process does not close after simulating browser crash. `KAJ-716`
+- Distant Reinitialization fails during Virus Scan. `KAK-1299`
```
[the remark this answers is not in the document]
Comment (pegnovi) [reply windows/CHANGELOG.md]: Fixed.

**Review by jfmoron — APPROVED** [review]

**Comment by jfmoron** [comment]
let's wait to do the merge that we are releasing and updating the date

**Review by brenthompson — CHANGES_REQUESTED** [review]
just looked at elux readme for another change i'm doing

**Review by jfmoron — APPROVED** [review]

**Pull request merged**